unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Bridge/SceneControllerBridge.cs

```diff
@@ -1,14 +1,18 @@
-using System.Collections;
-using DCL;
+using System;
 using UnityEngine;
+using Environment = DCL.Environment;
 
 public class SceneControllerBridge : MonoBehaviour
 {
     public void LoadParcelScenes(string payload) { Environment.i.world.sceneController.LoadParcelScenes(payload); }
 
     public void SendSceneMessage(string payload) { Environment.i.world.sceneController.SendSceneMessage(payload); }
-
-    public void UnloadScene(string sceneId) { Environment.i.world.sceneController.UnloadScene(sceneId);}
+    
+    // sceneNumber comes as a string because WebSocketCommunication can only receive strings as kernel message parameters
+    public void UnloadSceneV2(string sceneNumber)
+    {
+        Environment.i.world.sceneController.UnloadScene(Int32.Parse(sceneNumber));
```

Comment:
resilience improvement: use TryParse and add an error log in case it cannot be parsed

Reply:
done!